cfme/common/__init__.py

```diff
@@ -429,8 +429,11 @@ def get_tags(self, tenant="My Company Tags"):
         entities = view.entities
         if hasattr(entities, 'smart_management'):
             tag_table = entities.smart_management
+        elif hasattr(view.details, 'smart_mgmt'):
```

Comment:
I think it would be better to pick one name for this widget and stick with it. Moving all of the views to use ParametrizedSummaryTable is a bigger change, but I think that would be the optimal solution.

We're adding band-aids by doing this, and enabling poor consistency. While this change _works_, I would much much rather see work put in to make the views (and tests) consistent.

Reply:
If we're going to accept this as a temporary 'fix' for the test failure, I want to see a FIXME here at a minimum. But I would rather just address the non-consistent views _now_.

Reply:
I definitely agree with this, I will start working on ensuring consistency across our views and tests for tag_tables. :+1:

Reply:
Moving back to WIP since the PR will change significantly. Thanks @john-dupuy!